protopipe/pipeline/event_preparer.py

```diff
@@ -572,19 +572,31 @@ def prepare_event(self, source, return_stub=False, save_images=False, debug=Fals
                 with warnings.catch_warnings():
                     warnings.simplefilter("ignore")
 
+                    tels_pointing = {
+                        tel_id: SkyCoord(
+                            alt=point_altitude_dict[tel_id],
+                            az=point_azimuth_dict[tel_id],
+                            frame="altaz",
+                        )  # cycle only on tels which still have an image
+                        for tel_id in point_altitude_dict.keys()
+                    }
+
+                    # if all telescopes are pointing in parallel to run_array_direction, no need to pass the tels
+                    # pointing to predict
+                    for tel_id in tels_pointing:
+                        if tels_pointing[tel_id].alt != event.mcheader.run_array_direction[1]:
+                            tels_pointing = None
```

Comment:
should you really set your loop variable to None inside the loop? That seems to be bad form.  This is just checking that the telescopes are parallel right? I would suggest either using something like 
```py3
all(x.alt==dir[1] and x.az==dir[0] for x in tels_pointing.values())
```
 or just dropping this case.  In real data, there will always be a mispointing, so generally we never have it perfectly equal (and of course this check only works if this is mcdata anyhow).

Reply:
could we fix this? It shouldn't change anything for parallel simulations, but it would solve #17 for divergent pointing